src/Providers/Stripe/Token.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace PodPoint\Payments\Providers\Stripe;
+
+class Token extends \PodPoint\Payments\Token
```

Comment:
Is there a reason you use a direct import instead of using a `use` statement?

Reply:
Not particularly - would have to import at `BaseToken` or something which I aint overly fond of